src/tests/GC/API/GC/GetTotalPauseDuration.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// Tests GC.GetTotalPauseDuration()
+
+using System;
+using System.Diagnostics;
+
+public class Test_Collect
```

Comment:
this test seems a little thin. I would at least call the GetGCMemoryInfo API and verify that it agrees with that result.